coverage/cmdline.py

```diff
@@ -227,7 +229,15 @@ class Opts:
         "", "--version", action="store_true",
         help="Display version information and exit.",
     )
-
+    dump_signal = optparse.make_option(
```

Comment:
Overall, I think this should be called "save signal" instead of "dump signal" throughout.

Reply:
These make_option definitions are in rough alphabetical order, so this should be further up.